src/wp-includes/media.php

```diff
@@ -1963,6 +1975,63 @@ function wp_filter_content_tags( $content, $context = null ) {
 	return $content;
 }
 
+/**
+ * Adds auto to the sizes attribute to the image, if image lazy loaded.
+ *
+ * @since 6.7.0
+ *
+ * @param string|mixed $html The HTML image tag markup being filtered.
```

Comment:
I think this should be documented as always being a `string`, similar to other functions used in the same way, e.g. `wp_img_tag_add_loading_optimization_attrs()`.
```suggestion
 * @param string $html The HTML image tag markup being filtered.
```